web/templates/profile/show.html.eex

```diff
@@ -23,8 +23,8 @@
     </div>
   </div>
   <div class="col-md-7 col-sm-12">
-    <h3>Member of</h3>
     <%= for group <- @user.groups do %>
+      <h3>Member of</h3>
```

Comment:
D'oh. Ya I was testing on a single group, this didn't cross my mind. Good catch.